package.json

```diff
@@ -107,8 +107,8 @@
 		"rules": {
 			"no-alert": "off",
 			"import/no-unassigned-import": "off",
-			"import/no-unresolved": "warn",
-			"import/named": "warn"
+			"import/no-unresolved": "off",
+			"import/named": "off"
```

Comment:
Reportedly disabled upstream: https://github.com/xojs/xo/commit/ee145cbe719073aeccd349af4bb85ddc542a2818

...but it appears to still be enabled here, so I'm disabling it manually.

Reply:
Not released yet.